update.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "v1.2",
+  "versionCode": 3,
+  "zipUrl": "https://github.com/AdguardTeam/adguardcert/releases/download/v1.2/adguardcert-v1.2.zip",
```

Comment:
Should point to `latest`, probably.

Reply:
It is better to have exact version in manifest